src/background/modules/ContextMenu.js

```diff
@@ -61,6 +61,17 @@ function createItems() {
         contexts: ["page"]
     });
 
+    browser.runtime.onMessage.addListener((message) => {
```

Comment:
Okay, generally thougj let's move that into the `init` functions below, don't we? Otherwise it could get re-registered and triggered multiple times when `createItems()` is called?